ApiCalls.js

```diff
@@ -0,0 +1,11 @@
+export const fetchFlies = async () => {
+  try {
+    const response = await fetch(
+      "https://my-fly-box-api.herokuapp.com/api/v1/flies"
+    );
+    const flyData = await response.json();
+    return flyData;
+  } catch (error) {
+    console.error(error);
+  }
+};
```

Comment:
This is good that you had made the fetch calls in its own file.  Now I can't incorporate this file to break out the fetch call in the Add Fly Component.  Thanks for setting this up!